.circleci/config.yml

```diff
@@ -59,6 +59,21 @@ commands:
             curl -L --fail -o /tmp/kubectl https://storage.googleapis.com/kubernetes-release/release/v<< parameters.version >>/bin/$(uname -s | tr A-Z a-z)/amd64/kubectl
             sudo install /tmp/kubectl /usr/local/bin/kubectl
 
+  "install-helm":
+    parameters:
+      version:
+        type: string
+        default: "v3.5.4"
+    steps:
+      - run:
+          name: "Install Helm << parameters.version >>"
+          command: |
+            set -x
+            curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
+            chmod 700 get_helm.sh
+            ./get_helm.sh --version << parameters.version >>
+            rm ./get_helm.sh
+
```

Comment:
Since we're relying on `helm lint` in `make lint`, which is something we expect developers to run, I think I'd prefer to have this in `tools.mk`